docs/pages/enterprise/hsm.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: Teleport HSM Support
+description: How to configure Hardware Security Modules to manage your Teleport CA private keys
+h1: Teleport HSM Support
+---
+
+Teleport can be configured to use a connected HSM to create and hold CA private
+keys.
+
+While most PKCS#11 HSMs should be supported, we test with AWS CloudHSM,
+YubiHSM2, and SoftHSM2.
+
+# Setup
```

Comment:
Remove `#` and Setup in favor of prerequisites. Also we should not have `#` only `##`, as `#` is H1.